source/style-guide/release-notes-guidelines.txt

```diff
@@ -158,36 +158,34 @@ Use the following guideline when creating items in each section:
      - Use sentences to describe the new feature or enhancement.
        Provide as needed, and provide a link to any section in the
        documentation that describes that feature.
-     - IPv6 support has been added. When you create a server, an IPv4
-       address and an IPv6 address block are assigned. The IPv4 address and
-       IPv6 address block are used for the public interface (Internet). The
-       IPv4 address is also used for the private interface.
+     - Supports IPv6. When you create a server, you can specify 
+       both IPv4 and IPv6 addresses.
 
    * - Resolved issues
      - Provide an initial phrase that describes the issue that was
        fixed. Start the phrase with a past-tense verb. If necessary,
        include sentences to further explain the fix. If you list only
        phrases, do not use ending punctuation.
-     - Added support for Debian 8 (Jessie)
+     - - Supports Debian 8 (Jessie)
 
-       Trimmed the Monitoring ID and Monitoring Token configuration
-       variables to ensure correctness
+       - Trimms the Monitoring ID and Monitoring Token configuration
+         variables to ensure correctness.
 
-       Fixed the Xen Server 6 package repo
+       - Fixes the Xen Server 6 package repo.
 
    * - Known issues
      - Use sentences to describe the issue. If a workaround is
        available, explain it.
-     - The create a service operation does not support Classless
-       Inter-Domain Routing (CIDR) for IP address restrictions.
+     - Results from the ``$indexStats`` operator exclude 
+       queries that use the ``$match`` or ``mapReduce`` functions.
 
    * - Documentation changes
      - Provide an initial phrase that describes the issue that was
        fixed. Start the phrase with a past-tense verb. If necessary,
        include sentences to further explain the fix. If you list only
        phrases, do not use ending punctuation. Provide a link to the
        relevant section in the documentation.
-     - Clarified the descriptions of the alert policies to include how
+     - Clarifies the alert policy descriptions to include how
```

Comment:
Issue:
The existing and revised example don't match the guidelines.

Suggestions:

1) Fix the guidance to match the examples.

2) Personally, I would interpret "Documentation changes" as "what in the documentation changed?" which would  match with the existing past tense guidance. However, I'd be fine with present tense if it works better for everyone.

Reply:
@ccho-mongodb, I think of it more like "this release..."